pkg/framework/plugins/removepodsviolatingtopologyspreadconstraint/topologyspreadconstraint_test.go

```diff
@@ -371,8 +374,10 @@ func TestTopologySpreadConstraint(t *testing.T) {
 				},
 			}),
 			expectedEvictedCount: 66,
-			strategy:             api.DeschedulerStrategy{},
-			namespaces:           []string{"ns1"},
+			strategy: api.DeschedulerStrategy{
```

Comment:
wouldn't it be better to just pass in `componentconfig.RemovePodsViolatingTopologySpreadConstraintArgs` as opposed to doing the mapping in the unit test?

Reply:
good call, i forgot to come back to this.